### PR TITLE
Social sharing

### PR DESCRIPTION
**Pull request by dericky1**
1. Added Facebook Share button using react-share
2. Also did a lot of typing/code structure cleanup for groups and profile page

### CODE REVIEW

File: fwb/app/(auth)/(routes)/fre3/page.tsx [hunk 1]
```diff
@@ -93,10 +95,6 @@ export default function UserFlowPage3() {
     window.open("https://www.instagram.com/");
   };
 
-  const handlefacebook = () => {
```
Comment (bohemiancode-x) [inline fwb/app/(auth)/(routes)/fre3/page.tsx:96]: Functions could be camel case? makes it easily readable. e.g handleFacebook

**Review by bohemiancode-x — COMMENTED** [review]
Great work on the share functions. Works perfectly. Tried inviting by inputting an email on FRE3 and that also worked. I got an invite in my email. (Although it rerouted to the profile page immediately without a success message, I think it's crucial to have one - maybe we can use react-toastify?). Also tried using a regular name (not email) for the FRE3 invite and that went through (should that be an error since it wasn't an email?)

File: fwb/app/(auth)/(routes)/fre3/page.tsx (hunk 1 above)
Comment (dericky1) [reply fwb/app/(auth)/(routes)/fre3/page.tsx:96]: Updated the naming.

**Comment by bohemiancode-x** [comment]
> > Great work on the share functions. Works perfectly. Tried inviting by inputting an email on FRE3 and that also worked. I got an invite in my email. (Although it rerouted to the profile page immediately without a success message, I think it's crucial to have one - maybe we can use react-toastify?). Also tried using a regular name (not email) for the FRE3 invite and that went through (should that be an error since it wasn't an email?)
> 
> Yeah, we should have a success message for sure. Could add that to the backlog. Just to clarify I only worked on the share feature for when you press the Facebook icon on /fre3. I haven't tested the share with an email.

okay, solid! well that feature lgtm!

**Review by bohemiancode-x — APPROVED** [review]

**Pull request merged**